cmd/juju/status/status_internal_test.go

```diff
@@ -6099,6 +6099,7 @@ func (s *StatusSuite) PrepareBranchesOutput(c *gc.C) *context {
 }
 
 func (s *StatusSuite) TestBranchesOutputTabular(c *gc.C) {
+	c.Skip("bug: #1862376 - can't always read our own writes. Model-cache may be too slow")
```

Comment:
this should be further discussed in a sprint. -> How do we want to handle test which rely on the state, hence modelcache.
If the modelcache is sometimes to slow. Do we have some kind of wait?